src/components/WebMapView.tsx

```diff
@@ -6,22 +6,20 @@ const WebMapView = () => {
   const mapRef = useRef<HTMLDivElement>(null);
   const { setState } = useContext(AppContext);
   
-  let mounted = false;
   // we can let the application
   // context know that the component
   // is mounted and ready
   useEffect(
     () => {
-      mounted = true;
       setState({
-        mounted,
         container: mapRef.current
       });
-      return () => {
-        mounted = false;
-      };
+      // TODO: clean up function?
+      // only thing I could think of here would be to set mapRef.current = null
+      // in the hopes that would end up causing view.container to be set to null
     },
-    [ mapRef.current ]
+    // NOTE: this ensures this is only fired once, like componentDidMount()
+    []
```

Comment:
I noticed this effect was running twice w/ the same value, so I think it's better to ensure it runs only on mount